ui-ngx/src/app/modules/home/pages/widget/select-widget-type-dialog.component.html

```diff
@@ -19,6 +19,7 @@
   <mat-toolbar color="primary">
     <h2 translate>widget.select-widget-type</h2>
     <span class="flex-1"></span>
+    <div [tb-help]="widgetTypesHelpLink"></div>
```

Comment:
It can be simplified `tb-help="widgetTypes"`

Reply:
it can be done with `tb-help="{{ widgetTypesHelpLink }}"`, but it basically the same